packages/qwik/src/optimizer/src/qwik-binding-map.ts

```diff
@@ -30,15 +30,5 @@ export const QWIK_BINDING_MAP = {
         "platformArchABI": "qwik.win32-x64-msvc.node"
       }
     ]
-  },
```

Comment:
Don't remove this

Reply:
I see. This might have been automatically modified by `pnpm run build.full`.
I will manually revert this, but it could solve?
I am a bit confusion about the current build process has changed from before.